docs/user-guide/api-mediation/configuration-unique-cookie-name-for-multiple-zowe-instances.md

```diff
@@ -8,11 +8,14 @@ To prevent overwriting of the default cookie name in the case of multiple Zowe i
 
 Follow this procedure to configure a unique cookie name for the instances:
 
-1. Open the `zowe.yaml` configuration file.
-2. Find or add the property  `components.gateway.apiml.security.auth.uniqueCookie`, and set it to `true`. A unique cookie name is generated as `apimlAuthenticationToken.cookieIdentifier`.
+1. Open the Zowe YAML configuration file.
+2. Find or add the property  `components.gateway.apiml.security.auth.uniqueCookie`, and set it to `true`.
+3. Find the property `zowe.cookieIdentifier` and set it to any word or number that you wish to identify your Zowe instance.
+4. Restart Zowe.
+
+This will set a suffix for the API Gateway's cookie name based upon the value of `zowe.cookieIdentifier`.
 
     **Example:**  
-    If this parameter is set to `true`, and the cookieIdentifier is `1`, the name of the cookie transforms to `apimlAuthenticationToken.1`.  
-    If this property is not set to `true`, the cookie name remains `apimlAuthenticationToken` by default.
+    If `components.gateway.apiml.security.auth.uniqueCookie` parameter is set to `true` and `zowe.cookieIdentifier` is `1`, the cookie name will be `apimlAuthenticationToken.1`.  
```

Comment:
For the live site and previous versions:
- Please add a definite article: "If **the** `bbbb` parameter is set ... "
- Avoid future tense: " ... the cookie name is `blahblah`."